sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/RecognizeReceiptsFromUrl.java

```diff
@@ -23,80 +23,89 @@
      *
      * @param args Unused. Arguments to the program.
      */
+    @SuppressWarnings("unchecked")
     public static void main(final String[] args) {
         // Instantiate a client that will be used to call the service.
         FormRecognizerClient client = new FormRecognizerClientBuilder()
             .credential(new AzureKeyCredential("{key}"))
             .endpoint("https://{endpoint}.cognitiveservices.azure.com/")
             .buildClient();
 
-        String receiptUrl = "https://raw.githubusercontent.com/Azure/azure-sdk-for-java/master/sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/sample-forms/receipts/contoso-allinone.jpg";
-        SyncPoller<OperationResult, List<RecognizedReceipt>> recognizeReceiptPoller =
+        String receiptUrl = "https://raw.githubusercontent.com/Azure/azure-sdk-for-java/master/sdk/formrecognizer"
+            + "/azure-ai-formrecognizer/src/samples/java/sample-forms/receipts/contoso-allinone.jpg";
+        SyncPoller<OperationResult, List<RecognizedForm>> recognizeReceiptPoller =
             client.beginRecognizeReceiptsFromUrl(receiptUrl);
 
-        List<RecognizedReceipt> receiptPageResults = recognizeReceiptPoller.getFinalResult();
+        List<RecognizedForm> receiptPageResults = recognizeReceiptPoller.getFinalResult();
 
         for (int i = 0; i < receiptPageResults.size(); i++) {
-            RecognizedReceipt recognizedReceipt = receiptPageResults.get(i);
-            Map<String, FormField> recognizedFields = recognizedReceipt.getRecognizedForm().getFields();
+            RecognizedForm recognizedReceipt = receiptPageResults.get(i);
+            Map<String, FormField<?>> recognizedFields = recognizedReceipt.getFields();
             System.out.printf("----------- Recognized Receipt page %d -----------%n", i);
-            FormField merchantNameField = recognizedFields.get("MerchantName");
+            FormField<?> merchantNameField = recognizedFields.get("MerchantName");
             if (merchantNameField != null) {
-                if (merchantNameField.getFieldValue().getType() == FieldValueType.STRING) {
+                Object merchantNameFieldValue = merchantNameField.getValue();
+                if (merchantNameFieldValue instanceof String) {
```

Comment:
I feel like we need to either link to the API ref or somehow communicate the expected types for each receipt field

Reply:
added :)